src/Admin/AdherentAdmin.php

```diff
@@ -241,92 +227,66 @@ protected function configureFormFields(FormMapper $formMapper)
                 ->add('position', ActivityPositionType::class, [
                     'label' => 'Statut',
                 ])
-                ->add('mandates', ChoiceType::class, [
-                    'label' => 'adherent.mandate.admin.label',
-                    'choices' => Mandates::CHOICES,
-                    'required' => false,
-                    'multiple' => true,
-                ])
-            ->end()
-            ->with('Compte', ['class' => 'col-md-6'])
-                ->add('status', ChoiceType::class, [
-                    'label' => 'Etat du compte',
-                    'choices' => [
-                        'Activé' => Adherent::ENABLED,
-                        'Désactivé' => Adherent::DISABLED,
-                    ],
-                ])
                 ->add('subscriptionTypes', null, [
                     'label' => 'Abonné aux mails :',
                     'choice_label' => 'label',
                     'required' => false,
                     'multiple' => true,
                 ])
-            ->end()
-            ->with('Référent', ['class' => 'col-md-6'])
-                ->add('managedArea', ReferentManagedAreaType::class, [
-                    'label' => false,
-                    'required' => false,
-                ])
-            ->end()
-            ->with('Tags', ['class' => 'col-md-6'])
-                ->add('tags', 'sonata_type_model', [
-                    'label' => 'Tags admin',
-                    'multiple' => true,
-                    'by_reference' => false,
-                    'btn_add' => false,
+                ->add('media', null, [
+                    'label' => 'Photo',
                 ])
-            ->end()
-            ->with('Membre du Conseil', ['class' => 'col-md-6'])
-                ->add('boardMemberArea', ChoiceType::class, [
-                    'label' => 'Région',
-                    'choices' => BoardMember::AREAS_CHOICES,
+                ->add('description', TextareaType::class, [
+                    'label' => 'Biographie',
                     'required' => false,
-                    'mapped' => false,
-                    'help' => 'Laisser vide si l\'adhérent n\'est pas membre du Conseil.',
+                    'attr' => ['class' => 'content-editor', 'rows' => 20],
                 ])
-                ->add('boardMemberRoles', 'sonata_type_model', [
-                    'expanded' => true,
-                    'multiple' => true,
-                    'btn_add' => false,
-                    'class' => Role::class,
-                    'mapped' => false,
-                    'help' => 'Laisser vide si l\'adhérent n\'est pas membre du Conseil.',
+                ->add('twitterPageUrl', UrlType::class, [
+                    'label' => 'Page Twitter',
+                    'required' => false,
+                    'attr' => [
+                        'placeholder' => 'https://twitter.com/alexandredumoulin',
```

Comment:
Why this URL for the example ?
Maybe we can use this example syntax : `https://twitter.com/<username>` ?

Reply:
i had just kept the old integration from "Referent entity"
edited